pcsx2/gui/AppGameDatabase.cpp

```diff
@@ -139,8 +110,10 @@ void DBLoaderHelper::ReadGames()
 		if (!extractMultiLine()) extract();
 
 		if (!m_keyPair.IsOk()) continue;
-		if (m_keyPair.CompareKey(m_gamedb.getBaseKey()))
+		if (m_keyPair.CompareKey(m_gamedb.getBaseKey())) {
```

Comment:
I'd prefer if the bracket started in the next line under if. To keep it consistent with the rest of the code in the file.